docs/Makefile

```diff
@@ -59,6 +59,9 @@ html:
 livehtml:
 	sphinx-autobuild --port 8888 -j auto --watch _data -b dirhtml $(ALLSPHINXOPTS) $(BUILDDIR)/html
 
+dockerhtml:
```

Comment:
Worth commenting here that this is for use inside the Docker image, not externally to run it?

Reply:
Used solely to bind sphinx to the Docker Container - to do so requires deliberate --host "0.0.0.0" - will add a comment